var/spack/repos/builtin/packages/hiop/package.py

```diff
@@ -22,6 +22,7 @@ class Hiop(CMakePackage, CudaPackage, ROCmPackage):
     maintainers = ["ashermancinelli", "CameronRutherford", "pelesh"]
 
     # Most recent tagged snapshot is the preferred version when profiling.
+    version("0.7.0", commit="5f42ab34b419b7cf64d0fffb29d443b009dbfd75")
```

Comment:
```suggestion
    version("0.7.0", commit="5f42ab34b419b7cf64d0fffb29d443b009dbfd75", submodules=True)
```

@pelesh this should be all we need